extensions/binding/org.eclipse.smarthome.binding.tradfri/src/main/java/org/eclipse/smarthome/binding/tradfri/handler/TradfriLightHandler.java

```diff
@@ -226,7 +236,12 @@ private void handleBrightnessCommand(Command command) {
 
     private void handleColorTemperatureCommand(Command command) {
         if (command instanceof PercentType) {
-            setColorTemperature((PercentType) command);
+            PercentType newValue = (PercentType) command;
+            if (!state.getColorTemperature().equals(newValue)) {
```

Comment:
If we had sent some other Color command meanwhile, is it guaranteed that the `state.getColorTemperature()` is  changed? What value does it e.g. take if I send a "green" to the bulb?

Reply:
The state is re-created on each message received from the bridge so - it should be up-to-date

Reply:
@ivivanov-bg Color is supported now: https://github.com/eclipse/smarthome/pull/4271

Reply:
OK, Point taken, will check how it behaves tomorrow - but are you sure that the RGB light supports setting color temperature

Reply:
Yes